src/frontend/tests/playwright/test_e2e_user.py

```
@@ -333,8 +334,10 @@ def infinite_scroll_all_items(expected_total: int):
             assert initial_visible_count > 0
             final_visible_count = initial_visible_count
             for _ in range(6):
-                page.mouse.wheel(0, 5500)
-                page.wait_for_timeout(300)
+                load_more_button = page.locator("#infinite-scroll-trigger")
+                if load_more_button.count() == 0:
+                    break
+                with_htmx_wait(page, lambda: load_more_button.click())
```

Ruff (0.15.14)

[warning] Function definition does not bind loop variable `load_more_button`

(B023)